src/Microsoft.Health.Dicom.Functions/DurableFunctionsCleanup.cs

```diff
@@ -0,0 +1,55 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using EnsureThat;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using System.Threading;
+
+namespace Microsoft.Health.Dicom.Functions
+{
+    public static class DurableFunctionsCleanup
+    {
+        // Currently an HTTP trigger, TODO: change to the appropriate trigger.
+        [FunctionName("DurableFunctionsCleanup")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "get", "post", Route = null)] HttpRequest req,
+            [DurableClient] IDurableOrchestrationClient client,
+            ILogger log)
+        {
+            log.LogInformation("Durable Functions cleanup started.");
+
+            EnsureArg.IsNotNull(req, nameof(req));
+
+            EnsureArg.IsNotNull(client, nameof(client));
+
+            // Specify conditions for Function Queries.
+            OrchestrationStatusQueryCondition condition = new OrchestrationStatusQueryCondition();
```

Comment:
Don't we need to define some conditions for selecting instances? Because as of now we'll just purge whatever the default conditions return (which is a bit scary!)

Reply:
You may want to retrieve some of these conditions from a configuration, which means the function may need to be an instance method (non-`static`).